app/src/main/kotlin/cz/covid19cz/erouska/ui/main/MainActivity.kt

```diff
@@ -128,13 +132,15 @@ class MainActivity :
         customTabHelper.chromePackageName?.let {
             CustomTabsClient.bindCustomTabsService(this, it, customTabsConnection)
         }
+        prefs.setAppVisitedTimestamp()
     }
 
     override fun onStop() {
         if (connectedToCustomTabsService) {
             unbindService(customTabsConnection)
             connectedToCustomTabsService = false
         }
+        prefs.setAppVisitedTimestamp()
```

Comment:
Proč se to ukládá i v onStop?

Reply:
pretoze, ak by prisla notifikacia, zatial co je user v appke, tak ak by sa timestamp ukladal len v onStart, po odchode z appky by sa mu znovu na najblizsi check zobrazila notifikacia. takto je zaistena akakolvek interakcia s appkou po prijati notifikacie - bud otvorenie alebo zatvorenie.